XamarinCommunityToolkit/Actions/Snackbar/SnackbarArguments.shared.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace Xamarin.CommunityToolkit.Actions.Snackbar
```

Comment:
I am not sure about the namespace for this class.

Should this class be visible for the XCT consumers?
If not, probably we should put [EditorBrowasble] attribute over the class definition.